src/chat/UnreadNotice.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Animated, StyleSheet, Text, Easing } from 'react-native';
+import Icon from '../common/Icons';
+
+const styles = StyleSheet.create({
+  unreadContainer: {
+    padding: 2,
+    backgroundColor: '#96A3F9',
+    flexDirection: 'row',
+    position: 'absolute',
+    left: 0,
+    right: 0,
+    zIndex: 1,
+    alignItems: 'center'
+  },
+  unreadText: {
+    flex: 0.9,
+    color: '#FFFFFF',
+    fontSize: 14
+  },
+  icon: {
+    flex: 0.05,
+    width: 14,
+    height: 14,
+    margin: 8,
+    fontSize: 14,
+    color: 'white',
+    textAlign: 'center',
+  }
+  /*
+  crossIcon: {
+    fontSize: 16,
+    fontWeight: 'bold',
+    transform: [
+      { rotate: '45deg' }
+    ]
+  }
+  */
+});
+
+const POSITIONS = {
+  top: 'top',
+  bottom: 'bottom'
+};
+
+const showAnimationConfig = {
+  toValue: 1,
+  duration: 400,
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+};
+
+const hideAnimationConfig = {
+  toValue: 0,
+  duration: 400,
+  easing: Easing.bezier(0.17, 0.67, 0.11, 0.99)
+};
+
+export default class UnreadNotice extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      translateAnim: new Animated.Value(0)
+    };
+  }
+
+  componentDidMount() {
+    this.show();
+  }
+
+  hide = () => {
+    this.state.translateAnim.setValue(1);
+    Animated.timing(this.state.translateAnim, hideAnimationConfig).start();
+  };
+
+  show = () => {
+    this.state.translateAnim.setValue(0);
+    Animated.timing(this.state.translateAnim, showAnimationConfig).start();
+  };
+
+  dynamicContainerStyles = () => {
+    const { position } = this.props;
+    const translateMultiplier = position === POSITIONS.top ? -1 : 1;
+
+    return {
+      ...StyleSheet.flatten(styles.unreadContainer),
+      bottom: position === POSITIONS.bottom ? 0 : null,
+      top: position === POSITIONS.top ? 0 : null,
+      transform: [
+        {
+          translateY: this.state.translateAnim.interpolate({
+            inputRange: [0, 1],
+            outputRange: [translateMultiplier * 50, 0]
+          })
+        }
+      ]
+    };
+  };
+
+  render() {
+    return (
+      <Animated.View style={this.dynamicContainerStyles()}>
+        <Icon style={[styles.icon, styles.downArrowIcon]} name="md-arrow-down" />
```

Comment:
Have a look at `Icons.js` file. We use it to ensure we use only one icon type (since there are several Icon libs available), the file is self-explanatory.